mailhog-command.php

```diff
@@ -10,3 +10,18 @@
 }
 
 EE::add_command( 'mailhog', 'Mailhog_Command' );
+
+EE::add_hook('before_invoke:mailhog up', 'init_auth' );
```

Comment:
One more thing: You should be adding init code in `up` function directly https://github.com/EasyEngine/mailhog-command/blob/develop/src/Mailhog_Command.php#L33

This hooks should be used when we do not have access of events. like in other packages if you want to perform some operation before mailhog up command you can use hook.
But here you're already in this package, so you should directly execute the code in `up` function.